### DISCUSSION TITLE
kbe 是否支持 cellapp 之间的 AOI？

### DISCUSSION THREAD

**Discussion by yunjielu** [kbengine/kbengine#1476] [General]
例如 有两个 cellapp 分别管理了 不同的 cell
他们两个 cellapp 之间是否可以进行 AOI 的交叉互通？
例如 cellapp1 上的人能看到 cellapp2 上的人

**Comment by Wandar** [comment]
不支持

**Comment by 610789338** [comment]
支持的吧，kbe的特点不就是跨进程aoi么

**Reply by MacroGu** [reply]
不支持吧， 如果支持的话， 那就是已经实现了一个大场景切分给多个cellapp 管理，  显然这个功能还没有实现啊

**Comment by ybyaoyao** [comment]
我确定是支持的。
